app/Finalsite.xcodeproj/project.pbxproj

```diff
@@ -2250,7 +2250,7 @@
 				INFOPLIST_FILE = Jasonette/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = com.finalsite.admin;
+				PRODUCT_BUNDLE_IDENTIFIER = com.finalsite.finalsiteacademy2;
```

Comment:
wonder if we could do anything about these first couple of files to help us keep from pushing environment based changes